.yarn/versions/13f288ec.yml

```diff
@@ -0,0 +1,34 @@
+releases:
+  "@yarnpkg/core": minor
+  "@yarnpkg/plugin-essentials": minor
```

Comment:
The `@yarnpkg/cli` package also needs to be marked `minor`

Reply:
Thanks! Also I added this file to 63d1671

Because I found in [this reply](https://github.com/yarnpkg/berry/issues/1510#issuecomment-756671429) that I should have to add the version file to the main commit